lib/dream_html.ml

```diff
@@ -563,4 +563,5 @@ module Hx = struct
   let vals fmt = string_attr "data-hx-vals" fmt
   let ws_connect fmt = string_attr "data-ws-connect" fmt
   let ws_send = "data-ws-send", ""
+  let hs fmt = string_attr "_" fmt
```

Comment:
Could you reorder this in alphabetical position inside the module? Also, it needs the `~raw:true` argument as it is a script and we don't want to escape the contents.